File: src/python/entityfx/arithemtics_benchmark.py

from entityfx.arithmetics_base import ArithmeticsBase

class ArithemticsBenchmark(ArithmeticsBase):
    
    def benchImplementation(self) -> float:
        self._r = (0)
        i = 0
        while i < self._iterrations: 
            self._r += ArithmeticsBase._doArithmetics0(i)
            i += 1
        return self._r

File: src/dotnet/EntityFX.NetBenchamarks.Core/Generic/ArithemticsBenchmark.cs
using System.Diagnostics;

namespace EntityFX.NetBenchmark.Core.Generic
{
    public class ArithemticsBenchmark : ArithmeticsBase<double>, IBenchamrk
    {
        public ArithemticsBenchmark(IWriter writer) : base(writer)
        {
        }

        public override double BenchImplementation()
        {
            R = 0;
            for (long i = 0; i < Iterrations; i++)
            {
                R += DoArithmetics(i);
            }
            return R;
        }
    }
}
